publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/Publication.java

```diff
@@ -0,0 +1,143 @@
+package no.unit.nva.publication.doi.dto;
```

Comment:
PublicationMapper requires namespace to do its job to map the dao to a dto, due to converting identifier into a id. But I've removed the DAO constructor that were requested earlier. Doesnt make sense to have since we have ContributorMapper now.